src/index.css

```diff
@@ -1,27 +1,199 @@
 body {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
-
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+  background-color: white;
+}
+
+#root {
```

Comment:
- [ ] It would be good if the styling is the same as the one provided on Zeplin to meet the requirements. Kindly check some of the differences that are marked with black colour, and compare to see other differences. Use similar spacing and add the missing elements 👇
- Zeplin design
<img width="614" alt="Bookstore CMS" src="https://user-images.githubusercontent.com/46851023/151675567-d79e5e1c-60fa-4f21-a959-26d7ef91d551.png">

- Your design

<img width="960" alt="dire" src="https://user-images.githubusercontent.com/46851023/151675613-15ef935d-c435-4c9b-b2b1-8c58c2a189e2.png">